packages/yoroi-extension/app/components/wallet/transactions/TransactionRevamp.js

```diff
@@ -97,6 +103,62 @@ export default class TransactionRevamp extends Component<Props, State> {
     this.setState(prevState => ({ isExpanded: !prevState.isExpanded }));
   };
 
+  getTxType(data: WalletTransaction): string | null {
```

Comment:
I noticed that the `getTxType` and the `getTxTypeMsg` have quite similar logic! they are almost the same :) 
and they are used in the same place. I suggest updating `getTxTypeMsg` to return both **messages the status**. So the end result should be one function called `getTxType` that returns both.

```ts
function getTxType(intl: $npm$ReactIntl$IntlFormat, currency: string, data: WalletTransaction): [string, string | null] 
```

Reply:
Checking this, thanks for the feedback! :)